### PR TITLE
Issue v9 release

### LINKED ISSUE #1762: Update modules for v9
To prepare for v9 we need to go through the list of repositories in this issue and:

1. Determine if the module should be archived and marked as deprecated. Mostly we should look at the commit history to determine if the community is interested in the module. A secondary check would be inspecting the data for the module at [npm-stat.com](https://npm-stat.com/). Unless we have people using, and contributing, to modules, we should be reducing our maintenance burden by archiving them.
1. Evaluate if the module needs any updates. Most likely this will simply mean updating dependencies, dropping Node <= 16, and updating CI configuration accordingly. It would be nice to get the testing suite (`tap`) configured the same as in the core repo <strike>(we may need to skip this one: see below)</strike>, and any Coveralls usage removed, but this is not necessary for release.

If you are able to help with any of these, please create a new issue in the repo you're interested in that has the title "Updating for v9" and a body with something like "👋 I'm X, and would like to work on this." It should also include a link back to this issue. After that issue is created, either I (@jsumners) or @mcollina will create a `next` branch for you to target. Once that branch is created, you will be able to start a new Pull Request that targets it. We will help out with any questions you may have.

-----

+ [x] [json-colorizer](https://github.com/pinojs/json-colorizer) (@Ranieri93)
+ [x] [koa-pino-logger](https://github.com/pinojs/koa-pino-logger) (@big-kahuna-burger)
+ [ ] [pino-abstract-transport](https://github.com/pinojs/pino-abstract-transport)
+ [ ] [pino-arborsculpture](https://github.com/pinojs/pino-arborsculpture)
+ [ ] [pino-caller](https://github.com/pinojs/pino-caller)
+ [ ] [pino-clf](https://github.com/pinojs/pino-clf)
+ [ ] [pino-debug](https://github.com/pinojs/pino-debug)
+ [ ] [pino-elasticsearch](https://github.com/pinojs/pino-elasticsearch)
+ [ ] [pino-eventhub](https://github.com/pinojs/pino-eventhub)
+ [ ] [pino-filter](https://github.com/pinojs/pino-filter)
+ [ ] [pino-gelf](https://github.com/pinojs/pino-gelf)
+ [x] [pino-http](https://github.com/pinojs/pino-http)
+ [ ] [pino-http-print](https://github.com/pinojs/pino-http-print)
+ [ ] [pino-integration](https://github.com/pinojs/pino-integration)
+ [ ] [pino-mongodb](https://github.com/pinojs/pino-mongodb)
+ [ ] [pino-nextjs-example](https://github.com/pinojs/pino-nextjs-example)
+ [ ] [pino-noir](https://github.com/pinojs/pino-noir)
+ [x] [pino-pretty](https://github.com/pinojs/pino-pretty)
+ [ ] [pino-sapling](https://github.com/pinojs/pino-sapling)
+ [ ] [pino-socket](https://github.com/pinojs/pino-socket)
+ [ ] [pino-std-serializers](https://github.com/pinojs/pino-std-serializers)
+ [ ] [pino-syslog](https://github.com/pinojs/pino-syslog)
+ [ ] [pino-tee](https://github.com/pinojs/pino-tee)
+ [ ] [pino-toke](https://github.com/pinojs/pino-toke)
+ [ ] [pino-webpack-plugin](https://github.com/pinojs/pino-webpack-plugin)
+ [ ] [pinojs](https://github.com/pinojs/pinojs)
+ [ ] [quick-format-unescaped](https://github.com/pinojs/quick-format-unescaped)
+ [ ] [real-require](https://github.com/pinojs/real-require)
+ [ ] [restify-pino-logger](https://github.com/pinojs/restify-pino-logger)
+ [ ] [rill-pino-logger](https://github.com/pinojs/rill-pino-logger)
+ [x] [sonic-boom](https://github.com/pinojs/sonic-boom)
+ [x] [thread-stream](https://github.com/pinojs/thread-stream)

-----

**2023-09-16 Note about `tap`**

We may need to evaluate moving off of `tap` altogether. It is possible that Pino could one day be included in the OpenJSF, and at this time projects in that foundation cannot use BlueOak licensed dependencies (https://github.com/fastify/fastify-static/pull/406#pullrequestreview-1615330181). The latest version of `tap` (18) has changed to using the BlueOak license and seems there is no chance of it being dual licensed at MIT (https://github.com/tapjs/tapjs/issues/821#issuecomment-1721839895).

Any test suite updates should consider using https://nodejs.org/dist/latest-v18.x/docs/api/test.html instead.

See also https://github.com/openjs-foundation/cross-project-council/issues/1170

**2024-01-29 Further update on `tap`**

The licensing issue around `tap@18` has been resolved by the OSI approving Blue Oak (https://opensource.org/license/blue-oak-model-license/). We can continue to use `tap` without concern about the license conflicting with any future decision around Pino joining the OpenJSF.

### PR DESCRIPTION
**Pull request by jsumners**
Once merged and published, the LTS doc needs to be updated with new dates.

I wanted to resolve #1762 before doing this, but it's just too much work for (effectively) two people with a lot else going on. Basically, keeping up with all of the `next` branches and conflicts (I already need to update the work I did a few weeks ago on `pino-abstract-transport`) is out of our reach. I have come to the conclusion that we just need to issue v9, and then chip away at #1762. It'll be easier on us.

### CODE REVIEW

File: docs/lts.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ A "month" is defined as 30 consecutive days.
 
 | Version | Release Date | End Of LTS Date | Node.js              |
 | :------ | :----------- | :-------------- | :------------------- |
+| 9.x     | TBD.         | TBD             | 18, 20               |
 | 8.x     | 2022-06-01   | TBD             | 14, 16, 18           |
```
Comment (mcollina) [inline docs/lts.md:47]:
```suggestion
| 8.x     | 2022-06-01   | TBD             | 14, 16, 18, 20       |
```

**Review by mcollina — APPROVED** [review]
LGTM

I'll ship https://github.com/pinojs/pino/pull/1930 in a minor and then we can ship this. Ok?

**Comment by jsumners** [comment]
Sounds good. Ship this when you're ready or I'll get to it sometime after work.

**Comment by arthurfiorette** [comment]
What's the breaking change of v9?

**Comment by mcollina** [comment]
drop support of Node < latest v18.

**Comment by joostdecock** [comment]
> drop support of Node < latest v18.

Thanks, came here looking for this info, trying to figure out whether it was safe to upgrade.

The [release notes](https://github.com/pinojs/pino/releases/tag/v9.0.0) state:

> *Dropped support for Node.js 14, 16, & 19*

Which I found a bit confusing when trying to figure out whether v18 is still supported.

**Pull request merged**